packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -1240,6 +1241,24 @@ public ReactSelectionWatcher(ReactEditText editText) {
 
     @Override
     public void onSelectionChanged(int start, int end) {
+      // Calculate cursor position
+      Layout layout = mReactEditText.getLayout();
+      if (mReactEditText.getLayout() == null) {
+            mReactEditText.getViewTreeObserver().addOnGlobalLayoutListener(new ViewTreeObserver.OnGlobalLayoutListener() {
+                @Override
+                public void onGlobalLayout() {
+                    mReactEditText.getViewTreeObserver().removeOnGlobalLayoutListener(this);
+                    onSelectionChanged(start, end);
+                }
+            });
+            return;
+        }
```

Comment:
I observed that in version 0.71.6 of React Native, the mReactEditText variable can be null, so this is a hotfix to address that issue